src/main/java/com/dev/cinema/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -66,4 +66,11 @@ public MovieSession add(MovieSession movieSession) {
             }
         }
     }
+
+    @Override
+    public MovieSession getById(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            return session.get(MovieSession.class, id);
+        }
+    }
```

Comment:
In case something goes wrong in this function the error is ignored.
You'd better throw an exception in catch block here.

Reply:
error won't be ignored, it will be thrown. But you can add catch clock and wrap to other exception with adding some information. Since it's not a checked exception , it's up to you